scheduler/src/cook/mesos/api.clj

```diff
@@ -174,6 +175,13 @@
    (s/optional-key :extract) s/Bool
    (s/optional-key :cache) s/Bool})
 
+(def ComputeCluster
+  "Schema for a compute cluster"
+  {:compute-cluster-type s/Keyword
+   :compute-cluster-name s/Str
+   (s/optional-key :mesos-framework-id) s/Str
+   (s/optional-key :data-source) s/Keyword})
```

Comment:
Is this field just an artifact of the backfilling scheme? If so, let's just not backfill and drop it.

Reply:
Yes. Nuked.